_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title:  "The Future of Babel"
+author: Henry Zhu
+date:   2016-11-29 11:00:00
+categories: announcements
+share_text: "The Future of Babel"
+---
+
+How we're doing, where we want to go, and how you can help!
+
+> Previous issues: [Babel Roadmap #4130](https://github.com/babel/babel/issues/4130), [6.0 #2168](https://github.com/babel/babel/issues/2168)
+
+- Project Maintenance
+- Current Status: Where We are Now
+- Future: Where We Want to Go!
+
+## Project Maintenance
+
+Maintaining any large open source project could deserve its own article, but Babel itself covers a lot of ground: core packages, plugins, build integrations such as gulp, webpack, etc (many of which we ourselves may not use), and more. We want to be able to support community plugins, presets, and ideas.
+
+Babel has its place in many tutorials, guides, talks, etc for various frameworks. The project itself has definetely helped me to understand the language better through bug fixes and support questions. We are active on twitter, slack, and github across other projects that use Babel. We have to balance new features, bug fixes, browser support, speed of compilation/generated code, plugin options, ease of use, working with individuals, big companies, libraries, and more.
+
+I believe our main goal is to make it possible to write Javascript in any environment you support as seamless as possible and through that support the future of Javascript tooling.
```

Comment:
Required me to reread this sentence several times to understand. Perhaps rephrase.